fpl/cli.py

```diff
@@ -17,21 +17,22 @@ def download():
     blob_import.download_blobs_in_containers()
 
 
-@data.command(name="to-csv", help="Convert JSON to CSV")
+@data.command(name="csv_on_entity", help="Class to extract desired CSV files")
 @click.option(
     "--data-dir",
     "-d",
     type=click.Path(exists=True),
-    default="data/raw/2021-fpl-data",
+    default="data/raw/2020-fpl-data",
     help="Path to data-dir to transform",
 )
 @click.option(
     "--entity",
     "-e",
-    type=click.Choice(["elements", "teams"], case_sensitive=False),
+    type=click.Choice(["elements", "teams", "fixtures"], case_sensitive=False),
```

Comment:
Fixtures is a different dataset. This will not work as custom logic for fixtures dataset will be needed. Suggest this an exercise for the transformation increment.